src/renderers/webxr/WebXRManager.js

```diff
@@ -600,6 +603,12 @@ class WebXRManager extends EventDispatcher {
 
 		};
 
+		this.getKnownPlanes = function () {
```

Comment:
For simplicity, I would rename `getKnownPlanes()` to `getPlanes()`. Also the internal variables `knownPlanes` and `knownPlaneLastChangedTimes` could be just `planes` and `planesLastChangedTimes`.